### ISSUE TITLE
async/await error: Unexpected token function

### ISSUE THREAD

**Issue by JulianKingman** [meteor/promise#3]
I'm getting an 'Unexpected token' error when I try to use the async/await pattern. My code looks something like this, any idea what I'm doing wrong?

````javascript
Template.fragments.events({
    'submit .copyContent': async function(e,t){
    var text = await Meteor.call('getContent', selector);
    console.log(text.split"<p>");
});
````

**Comment by benjamn** [comment]
Yeah, `async` functions are currently not enabled by the `ecmascript` package, though I am eager to consider enabling them if there is interest.

**Comment by stubailo** [comment]
I'm interested!

**Comment by JulianKingman** [comment]
I’m obviously interested :)

I’ve been trying to understand how the example of the async/await pattern is used on okgrow:promise’s example: http://okgrow-promise.meteor.com/
I tried with grigio:babel, but it gave me the same error, and okgrow is not deprecated, so I assumed the same functions would be carried over into this package.

On September 16, 2015 at 1:07:55 AM, Sashko Stubailo (notifications@github.com) wrote:

I'm interested!

**Comment by lourd** [comment]
Thirded, interested!

**Comment by bySabi** [comment]
:+1:

**Comment by ShimShamSam** [comment]
Please enable this

**Comment by purplecones** [comment]
I'm also interested in this

**Comment by ShimShamSam** [comment]
@purplecones It's supported in Meteor 1.3+
